src/frontend/components/Felleskomponenter/SkjemaFeltInput/SkjemaFeltInput.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+import { Input, InputProps } from 'nav-frontend-skjema';
+
+import { Felt } from '@navikt/familie-skjema';
+
+import SpråkTekst from '../SpråkTekst/SpråkTekst';
+
+interface SkjemaFeltInputProps extends InputProps {
+    // eslint-disable-next-line
+    felt: Felt<any>;
+    visFeilmeldinger: boolean;
+    labelSpråkTekstId: string;
+}
+
+/**
+ * Henter input props fra felt, og fra props. Props overstyrer felt.
+ */
+export const SkjemaFeltInput: React.FC<SkjemaFeltInputProps> = props => {
+    const { felt, labelSpråkTekstId, visFeilmeldinger, ...navInputProps } = props;
+    return felt.erSynlig ? (
+        <Input
+            label={<SpråkTekst id={labelSpråkTekstId} />}
+            {...felt.hentNavInputProps(visFeilmeldinger)}
+            {...navInputProps}
```

Comment:
Hva er forskjellen på navInputProps og hentNavInputProps? Synes det er vanskelig å holde oversikt over hva som faktisk blir sendt rundt når det sendes med sånn "...props". Finnes det en måte å omgå det her?

Reply:
Skreiv en kommentar om akkurat det over her. `hentNavInputProps` henter ut stuff fra feltet, spesifikt `feil`, `id`, `onChange` og `value`, mens navInputProps er evt andre ting man kan sende til en nav-input, for eksempel `bredde`. `description`, `mini`, osv. Kan ikke omgå det uten å fjerne funksjonalitet fra nav-inputen, men er veldig åpen for forslag til annet navn på den variabelen.